src/serialization/writer.py

```diff
@@ -25,6 +25,29 @@
 flags.DEFINE_integer("numexamples", 1,
                      "Approx number of glyphs in each protobuf file")
 
+def notRepeat(glyph, fontDict):
+    lowercase =set(['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z'])
+    if glyph in lowercase:
```

Comment:
Will need to remove the next line of code but

```suggestion
    if glyph in lowercase and glyph.upper() in fontDict:
```